CHANGELOG-1.2.md

```diff
@@ -0,0 +1,92 @@
+# Release notes for 1.2.0
```

Comment:
Can we move all these into another folder similar to the other sidecars?

Reply:
done